### ISSUE TITLE
GTK4

### ISSUE THREAD

**Issue by travankor** [jmattheis/gruvbox-dark-gtk#3]
It would be nice to support gtk4 too. Some apps like easyeffects already use gtk4.

**Comment by jmattheis** [comment]
Yes, I think currently gtk4 isn't supported by the theme generator oomox I guess for the rest of the theme. So I don't really know how to add it (:.

**Comment by m-pasik** [comment]
I [tried](https://github.com/m-pasik/gruvbox-dark-gtk4/) (it's really bad but it sort of works)
